x/gamm/client/cli/tx.go

```diff
@@ -165,6 +166,16 @@ func NewExitSwapShareAmountIn() (*osmocli.TxCliDesc, *types.MsgExitSwapShareAmou
 	}, &types.MsgExitSwapShareAmountIn{}
 }
 
+func NewMigrateSharesToFullRangeConcentratedPosition() (*osmocli.TxCliDesc, *balancer.MsgMigrateSharesToFullRangeConcentratedPosition) {
+	cmd := &osmocli.TxCliDesc{
+		Use:              "migrate-position [shares-to-migrate]",
+		Short:            "migrate shares to full range concentrated position",
+		NumArgs:          1,
+		ParseAndBuildMsg: NewMigrateSharesToFullRangeConcentratedPositionMsg,
```

Comment:
We shouldn't need a custom parse function for this. Osmocli should do it all automatically. See tx.go in concentratedliquidity for examples